### ISSUE TITLE
Bug: Less does not recompile if imported files changed, even with force option

### ISSUE THREAD

**Issue by aDu** [emberfeather/less.js-middleware#156]
I have a file called **main.less**:
```@include "nav"```

And **nav.less**:
``` body {
    padding: 50px;
```

Express middleware:
```
app.use(lessMiddleware(__dirname + '/public', {
    force: true,
    debug: true
}))
```

Frontend includes only **main.css**:
```
<link rel="stylesheet" href="/css/main.css" />
```

When I run the website, it compiles nicely. But, when I update **nav.less**, those changes are not reflected (i.e. recompiled) on the website until I restart the express server. I disabled browser caching. However, if I update **main.less** it recompiles, but not on **nav.less**. In other words, recompiling does not work in `@include` files.

Using:
* Node v8.10.0.
* Less middleware v3.0.1.
* Express v4.16.3

**Comment by pigeonman99** [comment]
I faced the same issue. Downgrading to 2.2.1 fixed the problem.

**Comment by beaulac** [comment]
Was having the same issue; it is linked to an issue in less.js itself: https://github.com/less/less.js/issues/3253

This was fixed as of v3.5.0beta -- Downgrading to 2.2.1 of less-middleware works (because it uses 2.x, and apparently this bug was introduced somewhere in 3.x), but you could also do something like:
```
  "resolutions": {
    "less.js": "^3.9.0"
  }
```
so that less-middleware uses a fixed version of less.js.
